cob_bringup/package.xml

```diff
@@ -66,12 +62,8 @@
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>costmap_2d</exec_depend>
```

Comment:
not in cob_bringup, but in cob_hardware_config because we use it to specify a plugin type (see [here](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_hardware_config/cob4-2/config/local_costmap_params.yaml#L23))